packages/gatsby-theme-carbon/src/components/Video/Video.js

```diff
@@ -140,6 +140,9 @@ Video.propTypes = {
         `You can't specify children/tracks for vimeo videos. You can control the captions from the Vimeo control panel`
       );
     }
+    if (props.src && !props.poster) {
```

Comment:
Could you elaborate on the error here? A poster shouldn't be required. If we're getting an error from someone providing a source without a poster, we should fix that error rather than making posters required.

Reply:
I had some other issues with Gatsby that I think messed with this also, I just revered and that cleared the error that was happening here (gatsbyjs/gatsby#24860). I guess I can revert these changes and leave it as a doc update.